readme.md

```diff
@@ -82,7 +87,9 @@ var Panda = function(name, age) {
 
 Your Answer:
 ```text
-
+var eat_bamboo = function(){
+  num_bamboo_eaten ++;
+}
```

Comment:
This needs to be defined on the `Panda.prototype` so all instances of panda can use it, and it needs to use `this.num_bamboo_eaten` to refer to the property on that panda.